node_modules/oae-core/manageaccess/manageaccess.html

```diff
@@ -72,9 +76,12 @@ <h3 id="manageaccess-modal-title">__MSG__MANAGE_ACCESS__</h3>
                 {/for}
             </select>
         </div>
-        <button type="button" title="__MSG__REMOVE_MEMBER__" class="btn btn-link manageaccess-remove" data-id="${member.id}">
+        <button type="button" title="__MSG__REMOVE_MEMBER__" class="btn btn-link oae-listitem-remove" data-id="${member.id}">
             <i class="fa fa-times"><span class="sr-only">__MSG__REMOVE_MEMBER__</span></i>
         </button>
+        {if member.resourceType === 'email' && member.beenInvited}
+            <button type="button" title="__MSG__RESEND_INVITATION__" class="btn manageaccess-invitation-resend" data-email="${member.id|encodeForHTMLAttribute}">__MSG__RESEND__</button>
```

Comment:
We had discussed this layout while I was implementing it, and doing dynamic width of the select list based on whether or not a paper plane icon was there going to be too difficult to be worth it, so we had settled on this final implementation with screenshots.